lib/soccerama/client/commentaries.rb

```diff
@@ -0,0 +1,9 @@
+module Soccerama
+  class Client
+    module Commentaries
+      def commentaries(match_id)
+        get("/commentaries/match/#{match_id}")['commentaries']
```

Comment:
Agreed, at least we'll get a KeyError raised instead of getting nil if soccerama returned something unexpected.